src/EA.Weee.Web/Areas/Test/Controllers/CopyValuesController.cs

```diff
@@ -44,13 +44,13 @@ public ActionResult Paste(string returnUrl)
             return View("Paste", null, returnUrl);
         }
 
-        [HttpPost]
-        [ValidateAntiForgeryToken]
-        public ActionResult Paste(string pasteValues, string returnUrl)
-        {
-            TempData["pasteValues"] = pasteProcesser.BuildModel(pasteValues);
+        //[HttpPost]
+        //[ValidateAntiForgeryToken]
```

Comment:
Delete the Copy values controller in the test area now as we have an implemented version

Reply:
deleted